src/pages/pokemon/PokemonColection.tsx

```
@@ -0,0 +1,57 @@
+            ? "collection-container-active"
+            : "collection-container"
+        }
+      >
+        {viewDetail.isOpened ? (
+          <div className="overlay"></div>
+        ) : (
+          <div className=""></div>
+        )}
+        {pokemons.map((pokemon) => {
+          return (
+            <div onClick={() => selectPokemon(pokemon.id)}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)